torch/_prims/nvfuser_prims.py

```diff
@@ -291,6 +296,7 @@ def _amin_nvfuser(
 
 _nvfuser_impls["broadcast_in_dim"] = _broadcast_in_dim_nvfuser
 _nvfuser_impls["convert_element_type"] = _convert_element_type_nvfuser
+_nvfuser_impls["transpose"] = _transpose_nvfuser
```

Comment:
nitpick. we can use a knob to switch on/off fusion of transpose. Not that I'm questioning the performance of @zasdfgbnm 's transpose scheduler.... :rofl:

Reply:
If that would be needed we could add a special case to the partitioner code so that with a knob `nvprims.transpose` is either sent to nvfuser or eager.
It's easy to add.